RecoTracker/FinalTrackSelectors/plugins/TrackCutClassifier.cc

```diff
@@ -48,61 +66,157 @@ namespace {
   inline int nPixelHits(reco::Track const & tk) {
     return tk.hitPattern().numberOfValidPixelHits();
   }
-      
+  
   inline float dz(reco::Track const & trk, Point const & bestVertex) {
-      return std::abs(trk.dz(bestVertex));
+    return std::abs(trk.dz(bestVertex));
   }
   inline float dr(reco::Track const & trk, Point const & bestVertex) {
-      return std::abs(trk.dxy(bestVertex));
+    return std::abs(trk.dxy(bestVertex));
+  }
+
+  inline void dzCut_par1(reco::Track const & trk, int & nLayers, const float * par, float dzCut[]) {
+    float dzE =  trk.dzError();
+    for (int i=2; i>=0; --i) {
+      dzCut[i] = powN(par[i]*nLayers,4)*dzE;
+    }
+  }
+  inline void drCut_par1(reco::Track const & trk, int & nLayers, const float* par, float drCut[]) {
+    float drE =  trk.d0Error();
+    for (int i=2; i>=0; --i) {
+      drCut[i] = powN(par[i]*nLayers,4)*drE;
+    }
   }
+  
+  inline void dzCut_par2(reco::Track const & trk, int & nLayers, const float * par, float dzCut[]) {
+    float pt = float(trk.pt());
+    float eta = float(trk.eta());
 
+    // parametrized d0 resolution for the track pt
+    float nomd0E = sqrt(0.003*0.003+(0.001/pt)*(0.001/pt));
```

Comment:
I was hoping to use MVA and obsolete the use of Cuts.
I think that for phaseI at first order we can use Run2 MVA and then retrain MVA
I understand the need for a transition, in such a case I prefer to keep old MultiTrackSelector.
Still I urge to migrate the new PhaseI to a Run2 like configuration and use MVA.